demo/agilebank/demo.sh

```diff
@@ -100,6 +100,16 @@ echo
 
 pe "kubectl get k8sbannedimagetags -oyaml"
```

Comment:
k8sbannedimagetags shows audit results (and mostly exists to show the audit results). Is there something specific we're trying to solve with the extra commands?

Reply:
The k8sbannedimagetags example did not result in any violations (at least not in my test cluster) and we didnt create any deployment with latest tag images in the demo. Hence I added some thing that gives us violations.